plugins/modules/network/icx/icx_ping.py

```diff
@@ -6,9 +6,15 @@
 __metaclass__ = type
 
 
-DOCUMENTATION = '''
+ANSIBLE_METADATA = {'metadata_version': '1.1',
```

Comment:
Please make sure your PR only includes the bugfix, and doesn't destroy all improvements/fixes made to this module during the last months.

Also, all bugfix PRs need a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).